test/int/between_test.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:supercharged/supercharged.dart';
+
+void main() {
+  test("int between", () {
+    final int b = 1;
+    expect(() => b.between(null, 10), throwsArgumentError);
+    expect(() => b.between(10, null), throwsArgumentError);
+    expect(() => b.between(10, 6), throwsArgumentError);
```

Comment:
Lets make this scenario work. We can determine the lower and upper value inside the between functions.
For example:
```dart
var inhabitantsLondon = 9000000;
var inhabitantsParis = 2100000;

var myCity = 3000000;

myCity.between(inhabitantsLondon, inhabitantsParis);
```